resources/gui/layouts/game/CraftingWindow.json

```diff
@@ -152,7 +152,7 @@
         "Disabled": false,
         "Hidden": false,
         "RestrictToParent": false,
-        "MouseInputEnabled": true,
+        "MouseInputEnabled": false,
```

Comment:
did you worked this out with an updated asset file? this should be set as true according to #18